build.rs

```diff
@@ -19,20 +19,27 @@ fn main() -> Result<(), Box<dyn Error>> {
         println!("cargo:rustc-cfg=armv8m_main");
     } else if target.starts_with("armv7-") | target.starts_with("armv7a-") {
         println!("cargo:rustc-cfg=armv7a");
+    } else if target.starts_with("avr-") {
+        println!("cargo:rustc-cfg=avr");
     }
-
     // built-in targets with no atomic / CAS support as of nightly-2019-12-17
     // see the `no-atomics.sh` / `no-cas.sh` script sitting next to this file
     match &target[..] {
-        "msp430-none-elf" | "riscv32i-unknown-none-elf" | "riscv32imc-unknown-none-elf" => {}
+        "avr-atmega328p"
```

Comment:
I think this target name should be `avr-unknown-gnu-atmega328` based on the mentioned scripts

Reply:
Done.